my-new-folder/my-new-file.js

```diff
@@ -0,0 +1 @@
+console.log('this is my new branch');
```

Comment:
you should use `console.error` instead dude.

Reply:
I think you're the wrong dudeee!!!

Reply:
okay man